### PR TITLE
UI/homepage

### PR DESCRIPTION
**Pull request by maxellmilay**
Changes:
 - added homepage
 - made header fixed
 - added custom scrollbar
 - created API that can fetch fb page data and latest post
 - added website icon

Header:
image

Landing Section:
image

Social Media Bulletin:
image

Custom Scrollbar:
image

### CODE REVIEW

File: src/app/api/socmed/facebook/latest-post/route.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export async function GET() {
+  const fbPostRequest = await fetch(
+    `https://graph.facebook.com/gayangmingor/feed?access_token=${process.env.FB_PAGE_ACCESS_TOKEN}`,
+  );
+
+  const fbPostData: FbPostDataInterface = await fbPostRequest.json();
+
+  const imageRequest = await fetch(
+    `https://graph.facebook.com/${fbPostData.data[0].id}?fields=full_picture&access_token=${process.env.FB_PAGE_ACCESS_TOKEN}`,
```
Comment (ezerssss) [inline src/app/api/socmed/facebook/latest-post/route.ts]: transfer to a constant

File: src/app/api/socmed/facebook/latest-post/route.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  FbPostDataInterface,
+  FbPostImageDataInterface,
+  FbPostInterface,
+} from "@/interface/FbPostInterface";
+import { NextResponse } from "next/server";
+
+export async function GET() {
+  const fbPostRequest = await fetch(
+    `https://graph.facebook.com/gayangmingor/feed?access_token=${process.env.FB_PAGE_ACCESS_TOKEN}`,
+  );
```
Comment (ezerssss) [inline src/app/api/socmed/facebook/latest-post/route.ts]: constant

File: src/app/api/socmed/facebook/route.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import {
+  FbPageDataInterface,
+  FbPageImageDataInterface,
+  FbPageInterface,
+} from "@/interface/FbPageInterface";
+
+export async function GET() {
+  const pageRequest = await fetch(
+    `https://graph.facebook.com/gayangmingor?access_token=${process.env.FB_PAGE_ACCESS_TOKEN}`,
+  );
```
Comment (ezerssss) [inline src/app/api/socmed/facebook/route.ts]: constant

File: src/app/api/socmed/facebook/route.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  const pageData: FbPageDataInterface = await pageRequest.json();
+
+  const imageUrlRequest = await fetch(
+    "https://graph.facebook.com/gayangmingor/picture?redirect=0",
```
Comment (ezerssss) [inline src/app/api/socmed/facebook/route.ts]: constant or even enum

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: src/app/api/socmed/facebook/latest-post/route.ts (hunk 1 above)
Comment (maxellmilay) [reply src/app/api/socmed/facebook/latest-post/route.ts]: resolved

File: src/app/api/socmed/facebook/latest-post/route.ts (hunk 2 above)
Comment (maxellmilay) [reply src/app/api/socmed/facebook/latest-post/route.ts]: resolved

File: src/app/api/socmed/facebook/route.ts (hunk 3 above)
Comment (maxellmilay) [reply src/app/api/socmed/facebook/route.ts]: resolved

File: src/app/api/socmed/facebook/route.ts (hunk 4 above)
Comment (maxellmilay) [reply src/app/api/socmed/facebook/route.ts]: resolved

**Review by ezerssss — APPROVED** [review]

**Pull request merged**